src/options.cairo

```diff
@@ -200,18 +206,17 @@ mod Options {
         res
     }
 
-    fn run_add_0911_1611_options() {
+    fn run_add_0501_options() {
         let mut state = Governance::unsafe_new_contract_state();
         assert(!state.proposal_initializer_run.read(36), 'prop already initialized');
 
         state.proposal_initializer_run.write(36, true);
 
-        add_0911_options();
-        add_1611_options();
+        add_0501_options();
     }
 
-    fn add_0911_options() {
-        let MATURITY: felt252 = 1699574399;
+    fn add_0501_options() {
+        let MATURITY: felt252 = 1704499199;
```

Comment:
This is Fri Jan 05 2024 23:59:59 GMT+0000, but our standard option expiration time is Thu Nov 09 2023 23:59:59 GMT+0000

(So thursday). This is because options elsewhere expire friday morning.

Reply:
Should be fixed now, but please double check that the timestamp is correct now